sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -337,3 +337,13 @@ public partial class KeyVaultSettingsClient
         public override string ToString() { throw null; }
     }
 }
+namespace Azure.Security.KeyVault.Administration.Models
```

Comment:
Where are the new endpoints?

Reply:
Added them. Line 83-86:
https://github.com/Azure/azure-sdk-for-net/pull/43811/files#diff-6ffb7b435a28e7e1c9edcb87423a3fe952b67013a5ef958e7390212d8dc30783R83-R86